src/Draco.Compiler/Api/Scripting/ScriptingEngine.cs

```diff
@@ -11,17 +11,15 @@
 using Draco.Compiler.Internal.Codegen;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
+using static System.Net.Mime.MediaTypeNames;
 
 namespace Draco.Compiler.Api.Scripting;
 
 public static class ScriptingEngine
 {
-    public static void Execute(string text)
+    public static void Execute(string text, bool emitCS)
```

Comment:
No, let's not start dumping in more parameters that do random crap, let's plan out the API instead